clusters/build-clusters/01_cluster/openshift-machine-api/admin_build01-9hdwj-worker-us-east-1b_machine_set.yaml

```diff
@@ -30,17 +30,12 @@ spec:
           ami:
             id: ami-06f85a7940faa3217
           apiVersion: awsproviderconfig.openshift.io/v1beta1
-          blockDevices:
-          - ebs:
-              iops: 16000
-              volumeSize: 640
-              volumeType: io1
           credentialsSecret:
             name: aws-cloud-credentials
           deviceIndex: 0
           iamInstanceProfile:
             id: build01-9hdwj-worker-profile
-          instanceType: m5.4xlarge
+          instanceType: m5d.4xlarge
```

Comment:
Without the userDataSecret being updated, we won't get our new MCP

Reply:
Doing it right now.